nixos/modules/services/networking/mosquitto.nix

```diff
@@ -196,15 +195,14 @@ in
       wantedBy = [ "multi-user.target" ];
       after = [ "network.target" ];
       serviceConfig = {
-        Type = "forking";
         User = "mosquitto";
         Group = "mosquitto";
         RuntimeDirectory = "mosquitto";
         WorkingDirectory = cfg.dataDir;
         Restart = "on-failure";
-        ExecStart = "${pkgs.mosquitto}/bin/mosquitto -c ${mosquittoConf} -d";
+        ExecStart = "${pkgs.mosquitto}/bin/mosquitto -c ${mosquittoConf}";
         ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
-        PIDFile = "/run/mosquitto/pid";
+        PrivateTmp = true;
```

Comment:
What does it put into `/tmp`?

Reply:
I don't know if it does put anything there but it's one of those "just to be on the safe side" things. In any case, with ```DynamicUser = true;``` this is set anyway.